### ISSUE TITLE
listitem trigger onPress function & onLongPress function 

### ISSUE THREAD

**Issue by 25juan** [GeekyAnts/NativeBase#1946]
package.json
```
    "react": "16.2.0",
    "native-base": "^2.4.5",
    "react-native": "^0.53.3",
```

in ListItem's properties, add the onPress and onLongPress, when i tap the listitem ,the listitem trigger onPress and onLongPress events

my code
```
<ListItem
                button={true}
                onLongPress={()=>this.deleteModal(data)}
                onPress={()=>this.props.screenProps.navigation.navigate("ChatInput")}
                style={[styles.bordered,{marginLeft:18}]}
                avatar>
                <Left>
                    <Thumbnail source={ data.url} />
                </Left>
                <Body style={styles.none}>
                <Text style={styles.userName}>{data.name}</Text>
                <Text style={[subText,styles.subText1]} note>{data.text}</Text>
                </Body>
                <Right style={styles.none}>
                    <Text style={subText} note>{data.date}</Text>
                    <Text style={[subText,styles.subText2]} note>{data.place}</Text>
                </Right>
            </ListItem>

```

**Comment by akhil-geekyants** [comment]
@25juan checked this. It is working fine for me. Attaching a gif.

Code
```
import React, { Component } from 'react';
import { Container, Header, Content, List, ListItem, Left, Body, Right, Thumbnail, Text } from 'native-base';
export default class DynamicListExample extends Component {
  render() {
    var items = ['Simon Mignolet', 'Nathaniel Clyne', 'Dejan Lovren', 'Mama Sakho', 'Emre Can'];
    return (
      <Container>
        <Header />
        <Content>
          <List dataArray={items}
            renderRow={(item) =>
              <ListItem
                button={true}
                onLongPress={() => console.warn("event -> onLongPress")}
                onPress={() => console.warn("event -> onPress")}
                style={{ marginLeft: 18 }}
                avatar>
                <Left>
                  <Thumbnail source={{ uri: "https://nativebase.io/assets/img/front-page-icon.png" }} resizeMode="contain" />
                </Left>
                <Body>
                  <Text>Name</Text>
                  <Text note>Text</Text>
                </Body>
                <Right>
                  <Text note>Date</Text>
                  <Text note>Place</Text>
                </Right>
              </ListItem>
            }>
          </List>
        </Content >
      </Container >
    );
  }
}
```
Gif
![listitem](https://user-images.githubusercontent.com/27768184/40660056-9ade5ed6-636d-11e8-8356-0b1fc3eafc89.gif)

**Comment by 25juan** [comment]
in Android , listitem only trigger onLongPress events , i  wanna the delayLongPress  property

**Comment by 25juan** [comment]
ok ，thank.,

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.5.0](https://github.com/GeekyAnts/NativeBase/tree/v2.5.0)
